README.md

```diff
@@ -26,3 +26,23 @@ You'll need a running [blockchain-explorer-api](https://github.com/tari-project/
     
     # Serve from the docker container
     docker run -p 3000:3000 --env SERVE=1 --env-file=.env quay.io/tarilabs/block-explorer-frontend
+
+
+
+### To run locally without the setting up the back-end
+
+    # Change your env file
+    
+    REACT_APP_EXPLORER_API_DOMAIN=explore.tari.com/api
+    REACT_APP_EXPLORER_API_PREFIX=https
+    REACT_APP_EXPLORER_WS_PREFIX=ws
+    REACT_APP_TOKEN_NAME=tXTR
+
+    # Run
+    
+    $ npm run build
+    $ serve -s build
```

Comment:
Is this necessary? Does it not just work with `npm start`? I thought I tested it like that.. 

I don't think `serve` is a built-in unless the user has done `npm i -g serve`

Reply:
ooh need to check that - react told me to run serve -s build after I ran the build. Will take it out for now.

Reply:
So afaik, `react-scripts start` (which is the start script from `package.json` in this case) starts a server and continuously builds the source on changes, with hot module reloading. So the `build` step is actually just for deployment.